components/brave_user_agent/browser/DEPS

```diff
@@ -0,0 +1,3 @@
+include_rules = [
+  "+crypto",
+]
```

Comment:
EOF new line missing?

Reply:
Comments addressed in [cf7e398](https://github.com/brave/brave-core/pull/27620/commits/cf7e3984d70671391a7482466aea4a2d63f5c8d1)